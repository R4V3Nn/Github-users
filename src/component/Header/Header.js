import React from 'react';
import { browserHistory } from 'react-router';
import { HeaderBar } from '@talend/react-components';

const Header = () => {
	const brand = {
		id: 'brand',
		label: 'My App',
		onClick: () => browserHistory.push('/'),
	};
	return (
		<HeaderBar brand={brand} />
	);
};

export default Header;
