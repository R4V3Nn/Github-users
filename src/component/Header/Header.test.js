import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

describe('Header  component', () => {
	it('should render Header  component', () => {
		// when
		const wrapper = shallow(<Header />);

		// then
		expect(wrapper.getElement()).toMatchSnapshot();
	});
});
