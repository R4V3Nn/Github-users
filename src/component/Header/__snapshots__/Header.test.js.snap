// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header  component should render Header  component 1`] = `
<Translate(HeaderBar)
  brand={
    Object {
      "id": "brand",
      "label": "My App",
      "onClick": [Function],
    }
  }
/>
`;
